docs/user/installation_and_configuration.rst

```diff
@@ -347,7 +347,7 @@ Accumulo 1.6
 * accumulo-server-base-1.6.4.jar
 * accumulo-trace.1.6.4.jar
 * libthrift-0.9.1.jar
-* zookeeper-3.4.6.jar
+* zookeeper-3.4.5.jar
```

Comment:
We mention 3.4.5 elsewhere, and our main dependency is on 3.4.5.  I was just going for consistency.